### ISSUE TITLE
Static part of mask is removed multiple times from inserted value

### ISSUE THREAD

**Issue by Garretist** [RobinHerbots/Inputmask#1728]
I'm having issues with an input mask containing a static prefix in version 3.3.10.

I'm using the following mask (phone number): _07# ### ## ##_

It works fine until I try to insert 0 or 07 after "07" in the mask. When I insert only a 0 it will be removed, the same goes for 07, following this pattern everything is removed for 070, 0707, 07070 etc. as well. As far as I can tell the static prefix is repeatedly checked at the beginning of the inserted value and removed.

Examples of the result:
071 234 56 78 -> 071 234 56 78
070 123 45 67 -> 071 234 56 7
070 712 34 56 -> 071 234 56
070 701 23 45 -> 071 234 5

Am I missing some option to the plugin to remove this behavior?

**Comment by RobinHerbots** [comment]
@Garretist, 

Does this happen when you paste a value into it?

**Comment by Garretist** [comment]
Yes, both when I paste a value and when the plugin is initialized on the text box.

**Comment by RobinHerbots** [comment]
@Garretist, 

Thx for the feedback.

**Comment by Garretist** [comment]
Forgot to mention I added my own definition for #:
'#': {
	validator: "[\\d]",
	cardinality: 1,
}

**Comment by Garretist** [comment]
@RobinHerbots,

I believe I found the reason for my problems. Due to the way our system i built we managed to initialize the input mask several times and thus experienced the above behavior. Could not reproduce the problem when the plugin was only initialized once.

Maybe you would consider checking whether an input mask has already been initialized on the text box before initializing the same input mask again.

Sorry for sending you a false bug report but I'm very grateful you responded so quickly.
